packages/lesswrong/lib/collections/messages/newSchema.ts

```diff
@@ -97,6 +100,51 @@ const schema = {
       },
     },
   },
+  currentUserVote: DEFAULT_CURRENT_USER_VOTE_FIELD,
+  currentUserExtendedVote: DEFAULT_CURRENT_USER_EXTENDED_VOTE_FIELD,
+  allVotes: {
+    graphql: {
+      outputType: "[Vote!]",
+      canRead: ["guests"],
+      resolver: async (document, args, context) => {
+        const { currentUser } = context;
+        if (userIsAdminOrMod(currentUser)) {
+          return await getAllVotes(document, context);
+        } else {
+          return await getCurrentUserVotes(document, context);
+        }
+      },
+    },
+  },
+  voteCount: defaultVoteCountField('Messages'),
+  baseScore: {
+    ...DEFAULT_BASE_SCORE_FIELD,
+    graphql: {
+      ...DEFAULT_BASE_SCORE_FIELD.graphql,
+      outputType: "Float",
+    },
+  },
+  extendedScore: {
+    ...DEFAULT_EXTENDED_SCORE_FIELD,
+    graphql: {
+      ...DEFAULT_EXTENDED_SCORE_FIELD.graphql
+    },
+  },
+  score: DEFAULT_SCORE_FIELD,
+  inactive: DEFAULT_INACTIVE_FIELD,
+  afBaseScore: {
+    ...DEFAULT_AF_BASE_SCORE_FIELD,
+    graphql: {
+      ...DEFAULT_AF_BASE_SCORE_FIELD.graphql
+    },
+  },
+  afExtendedScore: {
+    ...DEFAULT_AF_EXTENDED_SCORE_FIELD,
+    graphql: {
+      ...DEFAULT_AF_EXTENDED_SCORE_FIELD.graphql
+    },
+  },
+  afVoteCount: DEFAULT_AF_VOTE_COUNT_FIELD,
```

Comment:
I assume, regrettably, that these 3 `af` fields are necessary because of all the logic in `voteServer` that assumes they exist